apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -160,6 +170,7 @@ export interface IRushConfigurationJson {
   projects: IRushConfigurationProjectJson[];
   eventHooks?: IEventHooksJson;
   hotfixChangeEnabled?: boolean;
+  packageManagerOptions?: IPackageManagerOptions;
```

Comment:
There are already `pnpmOptions` and `yarnOptions` objects. It seems like this should just be part of those.

Reply:
I thought this could be a place for settings that apply to the underlying package manager, irrespective of the specific choice made.

Reply:
I think it's confusing to have two fields which, on the surface level, appear to do the same thing. I think we should make this the base type of the other two fields.

@octogonz - do you have an opinion here?

Reply:
Interesting that the build is passing without changelogs for `rush-lib`

Reply:
`rush-lib`'s changelog is covered by `rush`'s changelog because they're in the same publishing group.

Reply:
>
>
> I think we should make this the base type of the other two fields.

Making this the base type in the rush.json would be a breaking change.